multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/cf/ClientFactory.java

```diff
@@ -10,19 +10,19 @@ public CloudControllerClient createClient(OAuth2AccessTokenWithAdditionalInfo to
         return createClient(createCredentials(token));
     }
 
-    public CloudControllerClient createClient(OAuth2AccessTokenWithAdditionalInfo token, String org, String space) {
-        return createClient(createCredentials(token), org, space);
+    public CloudControllerClient createClient(OAuth2AccessTokenWithAdditionalInfo token, String org, String space, String correlationId) {
```

Comment:
Can we prepare some builder where some of the values are required and others, as correlationid, not?
The direction where any new argument is added to all methods is not good.

Reply:
will think of a way